vnext/Shared/Modules/WebSocketModule.cpp

```diff
@@ -119,9 +149,24 @@ GetOrCreateWebSocket(int64_t id, string &&url, weak_ptr<WebSocketModule::SharedS
 
 namespace Microsoft::React {
 
-WebSocketModule::WebSocketModule() : m_sharedState{std::make_shared<SharedState>()} {
+#pragma region WebSocketModule
+
+WebSocketModule::WebSocketModule(winrt::Windows::Foundation::IInspectable const &inspectableProperties)
+    : m_sharedState{std::make_shared<SharedState>()},
+      m_proxy{std::make_shared<WebSocketModuleProxy>(inspectableProperties)} {
   m_sharedState->ResourceFactory = [](string &&url) { return IWebSocketResource::Make(); };
   m_sharedState->Module = this;
+  m_sharedState->InspectableProps = inspectableProperties;
+
+  auto propBag = ReactPropertyBag{m_sharedState->InspectableProps.try_as<IReactPropertyBag>()};
```

Comment:
>m_sharedState->InspectableProps [](http://example.com/codeflow?start=34&length=31)

WebSocketModuleTest create this module with "null" property bag

Reply:
Both this and Networking module should handle the scenario where the blobs are not available .. or the property bag is set to null ..

Reply:
The IInspectable arg should be guaranteed by InstanceImple (OInstance.cpp).